docs/bucket/notifications/README.md

```diff
@@ -479,7 +525,7 @@ With the `mc` tool, the configuration is very simple to add. Let us say that the
 # Create bucket named `images` in myminio
 mc mb myminio/images
 # Add notification configuration on the `images` bucket using the MySQL ARN. The --suffix argument filters events.
-mc events add  myminio/images arn:minio:sqs:us-east-1:1:postgresql --suffix .jpg
+mc events add myminio/images arn:minio:sqs:us-east-1:1:postgresql --suffix .jpg
```

Comment:
Not sure what needs to be done here. The doc has link to configuration about mysql/postgresql and those links have the updated information (updated in this PR).